src/redux/store.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers } from 'redux';
+import booksReducer from './books/booksSlice';
+import categoriesReducer from './categories/categoriesSlice';
+
+const rootReducer = combineReducers({
+  books: booksReducer,
+  categories: categoriesReducer,
+});
+
+const store = createStore(rootReducer);
+
+export default store;
```

Comment:
- [ ] You are using `redux-toolkit`, not regular `redux`. Please import `configureStore` from `@reduxjs/toolkit` and use it to create your store, instead of using the deprecated `createStore` function from `redux`.
[Hint] It will be like that:
```suggestion
import { configureStore } from "@reduxjs/toolkit"
import booksReducer from "./books/booksSlice"
import categoriesReducer from "./categories/categoriesSlice"

const store = configureStore({
  reducer: { books: booksReducer, categories: categoriesReducer },
})

export default store
```